types/webpagetest/index.d.ts

```diff
@@ -0,0 +1,407 @@
+// Type definitions for webpagetest 0.3
+// Project: https://www.npmjs.com/package/webpagetest
```

Comment:
It's better to use the code repository url: https://github.com/marcelduran/webpagetest-api